styles/components/avatar.css

```diff
@@ -0,0 +1,54 @@
+.avatar-holder{

```

Comment:
Try using avatar-container instead of holder. As it is more understandable.

Reply:
Sure will do that !